src/main/java/app/gpx_animator/Renderer.java

```diff
@@ -188,14 +211,7 @@ public void render(final RenderingContext rc) throws UserException {
             final Point2D marker = drawMarker(bi2, frame);
             if (font != null) {
                 drawInfo(bi2, frame, marker);
-
-                String att = resourceBundle.getString("configuration.attribution");
-                String getAt = cfg.getAttribution();
-                if (att.equals(getAt)) {
-                    drawAttribution(bi2, att.replace("%APPNAME_VERSION%", Constants.APPNAME_VERSION).replace("%MAP_ATTRIBUTION%", ""));
-                } else {
-                    drawAttribution(bi2, getAt);
-                }
+                drawAttribution(bi2, cfg.getAttribution());
```

Comment:
Looks like you removed the fix of bug #226. I guess this was a merge conflict that was resolved the wrong way.